lib/webhdfs/client_v1.rb

```diff
@@ -60,31 +62,65 @@ def initialize(host='localhost', port=50070, username=nil, doas=nil, proxy_addre
       @kerberos = false
       @kerberos_keytab = nil
       @http_headers = http_headers
+
+      @renew_delegation_token_time_hr = renew_token_hr
+      @d_token = nil
+
+      @latest_time = Time.now
```

Comment:
`@d_token` and `@latest_time` don't explain what those are. Use descriptive names, please.